test/unit/services/auto-approval/test-are-children-under-18.js

```diff
@@ -0,0 +1,65 @@
+const expect = require('chai').expect
+const moment = require('moment')
+
+const areChildrenUnder18 = require('../../../../app/services/auto-approval/checks/are-children-under-18')
+
+var autoApprovalDataClaimChildrenOver18 = {
+  claimChildren: [
+    {
+      ClaimChildId: 798118115,
+      EligibilityId: 798118115,
+      Reference: 'AUTOAPP',
+      ClaimId: 798118115,
+      Name: 'Sam Bloggs',
+      DateOfBirth: moment().subtract(17, 'years').toDate(),
```

Comment:
if the only property your checks looks at is DateOfBirth then you don't need the rest of this test data